### PR TITLE
Docs (quick fix): Multiple page templates

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?

This pull request introduces a "Before you begin" section. This section explains the steps to update your core package and synchronize the hCMS updates before using the templates.

### CODE REVIEW

File: apps/site/pages/docs/multiple-page-templates.mdx [hunk 1]
```diff
@@ -12,9 +12,29 @@ import { Callout } from 'nextra/components'
...
+
+1. Update your store’s `@faststore/core` package to the `v3.0.17` by running the following commands:
+
+```bash
+yarn add @faststore/core@3.0.17
+``` 
+```bash
+yarn add -D @faststore/cli@3.0.17
+```
+
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/multiple-page-templates.mdx:28]:
```suggestion
1. Update your store’s `@faststore/core` package to the `v3.0.17` by running the following command:

```bash
yarn add @faststore/core@3.0.17
``` 
```

File: apps/site/pages/docs/multiple-page-templates.mdx [hunk 2]
```diff
@@ -12,9 +12,29 @@ import { Callout } from 'nextra/components'
...
+To benefit from this feature, follow the steps below:
+
+1. Update your store’s `@faststore/core` package to the `v3.0.17` by running the following commands:
+
+```bash
+yarn add @faststore/core@3.0.17
+``` 
+```bash
+yarn add -D @faststore/cli@3.0.17
```
Comment (eduardoformiga) [inline apps/site/pages/docs/multiple-page-templates.mdx]: This feature specifically doesn't change the cli, and the cli has no 3.0.17 version. 
https://www.npmjs.com/package/@faststore/cli?activeTab=versions

So, I recommend using the most recent in this case.

File: apps/site/pages/docs/multiple-page-templates.mdx [hunk 3]
```diff
@@ -12,9 +12,29 @@ import { Callout } from 'nextra/components'
...
+```
+
+2. Run `yarn dev` to apply the changes to your project.
+
+3. After updating the `@faststore/core`, run the following command in your terminal to sync the changes from the Headless CMS:
+
+```bash
+faststore cms-sync
```
Comment (eduardoformiga) [inline apps/site/pages/docs/multiple-page-templates.mdx:29]:
```suggestion
yarn cms-sync
```

File: apps/site/pages/docs/multiple-page-templates.mdx (hunk 2 above)
Comment (Mariana-Caetano) [reply apps/site/pages/docs/multiple-page-templates.mdx]: Thanks! Fixed here: https://github.com/vtex/faststore/pull/2268/files#diff-05ed0a631665c06b7584525452a105b48461f81b9b3f816f7d2b346311b0163fR20-R28

**Review by eduardoformiga — APPROVED** [review]

**Pull request merged**